pkg/azure/core.go

```diff
@@ -346,5 +360,12 @@ func (d *MachinePlugin) GenerateMachineClassForMigration(ctx context.Context, re
 		return nil, status.Error(codes.Internal, "Migration cannot be done for this machineClass kind")
 	}
 
-	return &driver.GenerateMachineClassForMigrationResponse{}, fillUpMachineClass(azureMachineClass, req.MachineClass)
+	err := fillUpMachineClass(azureMachineClass, req.MachineClass)
+	if err == nil {
+		klog.V(2).Infof("MigrateMachineClass request has been processed successfully for %q", req.ClassSpec)
+	} else {
+		klog.V(2).Infof("MigrateMachineClass request has failed for %q, Error: %v", req.ClassSpec, err)
+	}
+
+	return &driver.GenerateMachineClassForMigrationResponse{}, err
```

Comment:
we can revert this.

Reply:
addressed in [d5d27b3](https://github.com/gardener/machine-controller-manager-provider-azure/pull/87/commits/d5d27b393645ba4258aaa137527b8c96fd095975)